File: solutions/python/leap/1/leap.py
def leap_year(year):
    return any((
                   divisible_by(year, 4) and not divisible_by(year, 100),
                   divisible_by(year, 100) and divisible_by(year, 400)
               ))
    
def divisible_by(number, divisor):
    return not number % divisor

File: solutions/clojure/leap/3/src/leap.clj
(ns leap)

(defn- divides? [number divisor]
  (zero? (mod number divisor))
)

(defn leap-year? [year] ;; <- argslist goes here
  (if (divides? year 100) (divides? year 400) (divides? year 4))
)
